crates/client-sdk/Cargo.toml

```
@@ -12,6 +12,9 @@ rust-version = { workspace = true }
 sdk = { workspace = true, default-features = false, features = [
     "full-model",
 ] }
+model = { path = "../hyle-model", package = "hyle-model", default-features = false, features = [
+    "full",
+] }
```

si tu veux malgré tout le garder en dépendance, fine, mais dans ce cas faut utiliser la version "workspace":

```suggestion
hyle-model = { workspace = true, default-features = false, features = [ "full" ] }
```

et autant retirer la feature "full-model" du sdk